server/enhancedDate.js

```diff
@@ -1 +1,126 @@
-// export this as a Node module using the enhancedDate.js code from Module_2
\ No newline at end of file
+var enhancedDate = function () {    
```

Comment:
Changed this to return an object with the 6 methods on it.  How do I add a newline at the end of the file?  Just carriage return?  How do I ask questions about the current version of the files I've pushed, instead of in the comments that you made on my outdated files?  Why does the enhancedDate.js file say that there are 2 lines not shown and only 3 lines in the file?  ...